.github/wiki/Getting-Started.md

```diff
@@ -44,6 +44,25 @@ cd() {
 check_directory_for_new_repository
 ```
 
+By @TheSast
+
+A fish adaptation of the previous script to put in your `$XDG_CONFIG_HOME/fish/functions/cd.fish`:
+```fish
+function cd -w='cd'
+  builtin cd $argv || return
+  check_directory_for_new_repository
+end
+
+function check_directory_for_new_repository
+  set current_repository (git rev-parse --show-toplevel 2> /dev/null)
```

Comment:
Should these be `set --local`?

Reply:
The variable is local to the currently executing function, I think `--local ` is used to restrict scope to the innermost block.
 AFAIK this should only be an issue if this was not inside a function, which it is,